### PR TITLE
Set header attr_reader on GraphCollection

### PR DESCRIPTION
**Pull request by jessieay**

### CODE REVIEW

**Comment by jessieay** [comment]
@jonny5

File: spec/cases/graph_api_spec.rb [hunk 1]
```diff
@@ -94,6 +94,38 @@
         expect(@api).to receive(:api).with(path, {}, 'get', {})
         @api.graph_call(path)
       end
+
+      describe "the http_component option" do
+        it "sends the http_component option to the API call" do
```
Comment (jonny5) [inline spec/cases/graph_api_spec.rb:99]: this test would've passed before changes right? basically just testing that the options get sent to api if passed in

File: spec/cases/graph_collection_spec.rb [hunk 2]
```diff
@@ -139,9 +149,20 @@
     it "returns a new GraphCollection of the result if it has an array data key and a paging key" do
       result = {"data" => [], "paging" => {}}
       expected = :foo
-      expect(Koala::Facebook::GraphCollection).to receive(:new).with(result, @api).and_return(expected)
+      expect(Koala::Facebook::GraphCollection).to receive(:new).with(result, @api, nil).and_return(expected)
       expect(Koala::Facebook::GraphCollection.evaluate(result, @api)).to eq(expected)
     end
+
+    context "headers passed in with request" do
```
Comment (jonny5) [inline spec/cases/graph_collection_spec.rb]: is this test necessary? above one seems like a better test

File: spec/cases/graph_api_spec.rb (hunk 1 above)
Comment (jessieay) [inline spec/cases/graph_api_spec.rb:99]: yes! just adding tests for options that weren't tested before

**Pull request merged**